store/siren-action-behavior.html

```diff
@@ -101,8 +101,14 @@
 				.then(function(resp) {
 					if (!resp.ok) {
 						var errMsg = resp.statusText + ' response executing ' + opts.method + ' on ' + href + '.';
-						var error = new Error(errMsg);
-						throw error;
+						if (resp.json) {
```

Comment:
The `json` method should always be available. I'm not sure how to check if the type is json parseable.

Reply:
Yeh, I'm surprised that the `json` method isn't always avaialble, but Emil seemed to be saying that in his testing it appeared it wasn't if the body didn't have a json representation. @emil-furniss can you double check that because according to the MDN docs it should always be there. It isn't clear from those docs thought what happens if the body is not parseable as JSON.

Also it seems like the result of calling `json()` should be an object, so it would seem strange that we would have to call `JSON.parse(data)` as it should already be parsed into an object.

Reply:
It wasn't clear to me from the docs if the `json()` method is always available - I'll remove that check if you say it is. 
And the result of calling `json()` here is a string, so we need `JSON.parse`. If the string is not parsable, that method will fail with a `SyntaxError`.